### PR TITLE
Updates for latest MOI

### PR DESCRIPTION
**Pull request by odow**
Remove dependence on MathOptInterfaceBridges now that it has been moved to MOI. 

Locally I'm getting a weird error. All the tests pass, an no error is thrown, but I get test errors...

From Travis, [it is a problem with the finalizer](https://travis-ci.org/JuliaOpt/Ipopt.jl/jobs/372586912#L4520-L4529).

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -1,82 +1,37 @@
...
+
+    # Test that the ipopt binary works
+    @test success(`$(Ipopt.amplexe) -v`)
+end
+
+@testset "MathProgBase" begin
+    include("MPBWrapper.jl")
+end
+
+@testset "MathOptInterface" begin
```
Comment (blegat) [inline test/runtests.jl]: Should we add an advice to do this in the guideline ?

File: test/runtests.jl [hunk 2]
```diff
@@ -21,7 +21,6 @@ using Ipopt, Base.Test
     # Test opening an output file
     openOutputFile(prob, "blah.txt", 5)
     # unlink the output file
-    Ipopt.freeProblem(prob)
```
Comment (odow) [inline test/runtests.jl]: I think I had to add that in order to free the lock on the output file.

Comment (mlubin) [reply test/runtests.jl]: Well that's no good because it calls `freeProblem` twice.

Comment (mlubin) [reply test/runtests.jl]: We can just add some logic in `freeProblem` to avoid this. But that can be a separate PR.

**Pull request merged**